src/magnetVacuumFF.cpp

```diff
@@ -31,155 +23,264 @@ using namespace dd4hep;
  * \endcode
  *
  */
+
+double getRotatedZ(double z, double x, double angle);
+double getRotatedX(double z, double x, double angle);
```

Comment:
These should be static to have static linkage, i.e. this function with this name is only valid within this file. Otherwise someone else could use getRotatedZ and pick up this one, or worse (for your case) you could end up using someone else's getRotatedZ whenever they implement one.

```suggestion
static double getRotatedZ(double z, double x, double angle);
static double getRotatedX(double z, double x, double angle);
```
and I would put the actual function implementation that you have below here as well.